cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-returned.cy.js

```diff
@@ -0,0 +1,247 @@
+          label: testData.userServicePoint.name,
+        },
+      ],
+    },
+    data: {},
+  };
+  const loanPolicyBody = {
+    id: uuid(),
+    name: `1_loan_${getRandomPostfix()}`,
+    loanable: true,
+    loansPolicy: {
+      closedLibraryDueDateManagementId: 'CURRENT_DUE_DATE_TIME',
+      period: {
+        duration: 1,
+        intervalId: 'Minutes',
+      },
+      profileId: 'Rolling',
+    },
+    renewable: true,
+    renewalsPolicy: {
+      unlimited: true,
+      renewFromId: 'CURRENT_DUE_DATE',
+    },
+  };
+
+  before('Preconditions', () => {
+    itemsData.itemsWithSeparateInstance.forEach(function (item, index) {
+      item.barcode = generateUniqueItemBarcodeWithShift(index);
+    });
+
+    cy.getAdminToken()
+      .then(() => {
+        ServicePoints.createViaApi(testData.userServicePoint);
+        testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((res) => {
+          testData.holdingTypeId = res[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((res) => {
+          testData.loanTypeId = res.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((res) => {
+          testData.materialTypeId = res.id;
+          testData.materialTypeName = res.name;
+        });
+      })
+      .then(() => {
+        itemsData.itemsWithSeparateInstance.forEach((item, index) => {
+          InventoryInstances.createFolioInstanceViaApi({
+            instance: {
+              instanceTypeId: testData.instanceTypeId,
+              title: item.instanceTitle,
+            },
+            holdings: [
+              {
+                holdingsTypeId: testData.holdingTypeId,
+                permanentLocationId: testData.defaultLocation.id,
+              },
+            ],
+            items: [
+              {
+                barcode: item.barcode,
+                status: { name: 'Available' },
+                permanentLoanType: { id: testData.loanTypeId },
+                materialType: { id: testData.materialTypeId },
+              },
+            ],
+          }).then((specialInstanceIds) => {
+            itemsData.itemsWithSeparateInstance[index].instanceId = specialInstanceIds.instanceId;
+            itemsData.itemsWithSeparateInstance[index].holdingId = specialInstanceIds.holdingIds[0].id;
+            itemsData.itemsWithSeparateInstance[index].itemId = specialInstanceIds.holdingIds[0].itemIds;
+          });
+        });
+        cy.wrap(itemsData.itemsWithSeparateInstance).as('items');
+      });
+
+    UsersOwners.createViaApi(owner.body).then((response) => {
+      owner.data = response;
+      PaymentMethods.createViaApi(response.id);
```

Comment:
I thought it will be automatically removed when we remove owner. In ui side it works like that but after you comment I checked it via api and get all payment methods and saw that it didn't remove. 

I added delete method in after block